docs/user/concepts.md

```diff
@@ -5,171 +5,145 @@ navOrder: 2
 
+- [FlowFuse Concepts](#flowfuse-concepts)
+  - [Table of Contents](#table-of-contents)
+  - [Team](#team)
+    - [Team Type](#team-type)
+  - [Application](#application)
+    - [DevOps Pipeline](#devops-pipeline)
+  - [Instance](#instance)
+    - [Hosted Instance](#hosted-instance)
+    - [Remote Instance](#remote-instance)
+    - [Instance Configuration](#instance-configuration)
+    - [Instance Type](#instance-type)
+    - [Stack](#stack)
+    - [Blueprint](#blueprint)
+    - [Template](#template)
+    - [Snapshot](#snapshot)
+      - [Hosted Instance Snapshot](#hosted-instance-snapshot)
+      - [Remote Instances (Device Snapshot)](#remote-instance-snapshot)
+  - [Device](#device)
+    - [Device Agent](#device-agent)
+    - [Fleet Mode vs Developer Mode](#fleet-mode-vs-developer-mode)
+    - [Provisioning Tokens](#provisioning-tokens)
+  - [Device Groups](#device-groups)
 
-### Team
+## Team
 
-Teams are how FlowFuse organises the user on the platform. Each team can have
-multiple members and each user can be a member of multiple teams.
+Teams are how FlowFuse organizes users on the platform. Each team can have multiple members and each user can be a member of multiple teams.
 
-The users in a team can have different roles that determine what they are
-[able to do](./team/#role-based-access-control).
+The users in a team can have different roles that determine what they are [able to do](./team/#role-based-access-control). In FlowFuse Cloud, each team has its own billing plan, managed via Stripe.
 
-In FlowFuse Cloud, each team has its own billing plan, managed via Stripe.
+### Team Type
 
-#### Team Type
+The platform can be configured to provide different types of teams. These can be used to apply limits on what teams of a given type can do. For example, a particular team type may be restricted to certain types of [Node-RED instances](#instance), or how many members the team can have.
 
-The platform can be configured to provide different types of teams. These can be used
-to apply limits on what teams of a given type can do. For example, a particular
-team type may be restricted to certain types of [Node-RED Instance](#instance),
-or how many members the team can have.
-
-### Application
+## Application
 
 **Introduced in FlowFuse 1.5**
 
-To organise your Node-RED instances, they are grouped with Applications.
+To organize your Node-RED instances, they are grouped within Applications. With the 1.5 release, each Application has a single Node-RED instance. With the 1.6 release, an application can have multiple Node-RED instances.
 
-With the 1.5 release, each Application has a single Node-RED instance. With the 
-1.6 release, an application can have multiple Node-RED instances.
+Applications provide logical organization of related instances, support for DevOps pipeline workflows, simplified device group management, application-level audit logging, and clear organizational boundaries for managing multiple instances and devices.
```

Comment:
Might be worth adding a comment about Granular RBAC here as Applications can now also be an authorization boundry